apps/server/src/modules/video-conference/uc/video-conference.uc.ts

```diff
@@ -294,11 +306,40 @@ export class VideoConferenceUc {
 
 		const bbbResponse: BBBResponse<BBBBaseResponse> = await this.bbbService.end(config);
 
-		return {
+		return new VideoConferenceDTO<BBBBaseResponse>({
 			state: VideoConferenceState.FINISHED,
 			permission: PermissionMapping[bbbRole],
 			bbbResponse,
-		};
+		});
+	}
+
+	protected async isExpert(
```

Comment:
why based on role instead of permission?

Reply:
The requirement was to attach to the expert role. Of course we could have done it with a permission, but the permission would only depend on the expert role. I would not think of a scenario in which a permission would make more sense. Did you have something specific in mind?